e2e/show-certification-else.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect, type Page } from '@playwright/test';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto(
+    'http://localhost:8000/certification/certifieduser/responsive-web-design'
```

Comment:
@Fahdkassim Lets use the BaseUrl env variable here instead of hardcoding localhost.

Reply:
Done